### DISCUSSION TITLE
Nothing showing when web GUI launched.

### DISCUSSION THREAD

**Discussion by stratriderz** [JonathanTreffler/backblaze-personal-wine-container#147] [General]
When I launch the web GUI after installation I only see the following:
![image](https://github.com/JonathanTreffler/backblaze-personal-wine-container/assets/61076159/8a9babff-f3cf-45d3-abac-a33e16c2325f)
Am I missing something?
Container installed on Unraid with defaults other than changing the ports due to conflicts with other containers.

**Comment by JonathanTreffler** [comment]
Try the third entry in the troubleshooting section of the manual: https://github.com/JonathanTreffler/backblaze-personal-wine-container?tab=readme-ov-file#troubleshooting , that might be the issue.

You need to adapt the tip for Unraid, adding the two env variables (USER_ID=0, GROUP_ID=0) on Unraid is done through the GUI instead of the docker run command.

**Reply by stratriderz** [reply]
I changed the User ID and Group ID to 0 and 0 as suggested by the troubleshooting section reference and am seeing the same results.

![image](https://github.com/JonathanTreffler/backblaze-personal-wine-container/assets/61076159/b844c756-880c-4d30-9c7e-6848de385239)

Is it reaching out to Backblaze to download the client?

**Reply by JonathanTreffler** [reply]
no, it downloads Backblaze from archive.org.

Please send the logs of the container.

**Reply by stratriderz** [reply]
text  error  warn  system  array  login  

[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + fetch_and_install
[app         ] + cd /config/wine/dosdevices/c:/
[app         ] + '[' true = true ']'
[app         ] + log_message 'INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] ++ date
[app         ] + echo 'Mon 22 Apr 2024 07:02:39 PM UTC: INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] + curl -L https://www.backblaze.com/win32/install_backblaze.exe --output install_backblaze.exe
[app         ]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[app         ]                                  Dload  Upload   Total   Spent    Left  Speed
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]  41 28.7M   41 11.8M    0     0  10.9M      0  0:00:02  0:00:01  0:00:01 10.9M
[app         ] 100 28.7M  100 28.7M    0     0  14.8M      0  0:00:01  0:00:01 --:--:-- 19.8M
[app         ] + log_message 'INSTALLER: Starting install_backblaze.exe'
[app         ] ++ date
[app         ] + echo 'Mon 22 Apr 2024 07:02:41 PM UTC: INSTALLER: Starting install_backblaze.exe'
[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + WINEARCH=win64
[app         ] + WINEPREFIX=/config/wine/
[app         ] + wine64 install_backblaze.exe
[xvnc        ] Mon Apr 22 19:12:45 2024
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Mon Apr 22 19:12:49 2024
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 29
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 17 rects, 1.25968 Mpixels
[xvnc        ]  EncodeManager:            272 B (1:18525.5 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 4 rects, 465 pixels
[xvnc        ]  EncodeManager:                 178 B (1:10.7191 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 19 rects, 36.399 kpixels
[xvnc        ]  EncodeManager:                  2.15527 KiB (1:66.0734 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 2 rects, 2.24 kpixels
[xvnc        ]  EncodeManager:                  3.38184 KiB (1:2.59428 ratio)
[xvnc        ]  EncodeManager:   Total: 42 rects, 1.29879 Mpixels
[xvnc        ]  EncodeManager:          5.97656 KiB (1:848.963 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 685.384 kpixels in / 665.172 kpixels out
[xvnc        ]  ComparingUpdateTracker: (1:1.03039 ratio)
[xvnc        ] Mon Apr 22 19:16:03 2024
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 11
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 6 rects, 625.424 kpixels
[xvnc        ]  EncodeManager:            96 B (1:26060.1 ratio)
[xvnc        ]  EncodeManager:     Bitmap RLE: 1 rects, 4 pixels
[xvnc        ]  EncodeManager:                 23 B (1:1.21739 ratio)
[xvnc        ]  EncodeManager:     Indexed RLE: 8 rects, 20.018 kpixels
[xvnc        ]  EncodeManager:                  1.33496 KiB (1:58.6452 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 2 rects, 5.9 kpixels
[xvnc        ]  EncodeManager:                  5.52734 KiB (1:4.17385 ratio)
[xvnc        ]  EncodeManager:   Total: 17 rects, 651.346 kpixels
[xvnc        ]  EncodeManager:          6.97852 KiB (1:364.622 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 24.228 kpixels in / 16.616 kpixels out
[xvnc        ]  ComparingUpdateTracker: (1:1.45811 ratio)
[xvnc        ] Mon Apr 22 19:16:04 2024
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Mon Apr 22 19:17:21 2024
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888
[xvnc        ] Mon Apr 22 19:17:25 2024
[xvnc        ]  VNCSConnST:  closing /tmp/vnc.sock: Clean disconnection
[xvnc        ]  EncodeManager: Framebuffer updates: 2
[xvnc        ]  EncodeManager:   Tight:
[xvnc        ]  EncodeManager:     Solid: 2 rects, 625.42 kpixels
[xvnc        ]  EncodeManager:            32 B (1:78178.2 ratio)
[xvnc        ]  EncodeManager:   Tight (JPEG):
[xvnc        ]  EncodeManager:     Full Colour: 2 rects, 9.16 kpixels
[xvnc        ]  EncodeManager:                  9.29395 KiB (1:3.85247 ratio)
[xvnc        ]  EncodeManager:   Total: 4 rects, 634.58 kpixels
[xvnc        ]  EncodeManager:          9.3252 KiB (1:265.826 ratio)
[xvnc        ]  Connections: closed: /tmp/vnc.sock
[xvnc        ]  ComparingUpdateTracker: 0 pixels in / 0 pixels out
[xvnc        ]  ComparingUpdateTracker: (1:-nan ratio)

**Reply by stratriderz** [reply]
I restarted the container and grabbed the logs:


text  error  warn  system  array  login  

[supervisor  ] loading service 'logrotate'...
[supervisor  ] all services loaded.
[supervisor  ] starting services...
[supervisor  ] starting service 'xvnc'...
[xvnc        ] Xvnc TigerVNC 1.13.1 - built Dec 21 2023 00:39:59
[xvnc        ] Copyright (C) 1999-2022 TigerVNC Team and many others (see README.rst)
[xvnc        ] See https://www.tigervnc.org for information on TigerVNC.
[xvnc        ] Underlying X server release 12014000
[xvnc        ] Mon Apr 22 19:27:55 2024
[xvnc        ]  vncext:      VNC extension running!
[xvnc        ]  vncext:      Listening for VNC connections on /tmp/vnc.sock (mode 0660)
[xvnc        ]  vncext:      Listening for VNC connections on all interface(s), port 5900
[xvnc        ]  vncext:      created VNC server for screen 0
[supervisor  ] starting service 'nginx'...
[nginx       ] Listening for HTTP connections on port 5800.
[supervisor  ] starting service 'openbox'...
[supervisor  ] starting service 'app'...
[app         ] + local_version_file=/config/wine/dosdevices/c:/ProgramData/Backblaze/bzdata/bzreports/bzserv_version.txt
[app         ] + install_exe_path=/config/wine/dosdevices/c:/
[app         ] + log_file=/config/wine/dosdevices/c:/backblaze-wine-startapp.log
[app         ] + custom_user_agent='backblaze-personal-wine (JonathanTreffler, +https://github.com/JonathanTreffler/backblaze-personal-wine-container), CFNetwork'
[app         ] + pinned_bz_version_file=/PINNED_BZ_VERSION
[app         ] ++ sed -n 1p /PINNED_BZ_VERSION
[app         ] + pinned_bz_version=9.0.1.767
[app         ] ++ sed -n 2p /PINNED_BZ_VERSION
[app         ] + pinned_bz_version_url=https://web.archive.org/web/20240224185620/https://secure.backblaze.com/win32/install_backblaze.exe
[app         ] + export WINEARCH=win64
[app         ] + WINEARCH=win64
[app         ] + export WINEDLLOVERRIDES=mscoree=
[app         ] + WINEDLLOVERRIDES=mscoree=
[app         ] + '[' '!' -f /config/wine/system.reg ']'
[app         ] + for x in {d..z}
[app         ] + test -d /drive_d
[app         ] + for x in {d..z}
[app         ] + test -d /drive_e
[app         ] + for x in {d..z}
[app         ] + test -d /drive_f
[app         ] + for x in {d..z}
[app         ] + test -d /drive_g
[app         ] + for x in {d..z}
[app         ] + test -d /drive_h
[app         ] + for x in {d..z}
[app         ] + test -d /drive_i
[app         ] + for x in {d..z}
[app         ] + test -d /drive_j
[app         ] + for x in {d..z}
[app         ] + test -d /drive_k
[app         ] + for x in {d..z}
[app         ] + test -d /drive_l
[app         ] + for x in {d..z}
[app         ] + test -d /drive_m
[app         ] + for x in {d..z}
[app         ] + test -d /drive_n
[app         ] + for x in {d..z}
[app         ] + test -d /drive_o
[app         ] + for x in {d..z}
[app         ] + test -d /drive_p
[app         ] + for x in {d..z}
[app         ] + test -d /drive_q
[app         ] + for x in {d..z}
[app         ] + test -d /drive_r
[app         ] + for x in {d..z}
[app         ] + test -d /drive_s
[app         ] + for x in {d..z}
[app         ] + test -d /drive_t
[app         ] + for x in {d..z}
[app         ] + test -d /drive_u
[app         ] + for x in {d..z}
[app         ] + test -d /drive_v
[app         ] + for x in {d..z}
[app         ] + test -d /drive_w
[app         ] + for x in {d..z}
[app         ] + test -d /drive_x
[app         ] + for x in {d..z}
[app         ] + test -d /drive_y
[app         ] + for x in {d..z}
[app         ] + test -d /drive_z
[app         ] + cd /config/wine/
[app         ] + '[' false = true ']'
[app         ] + '[' -n 900 ']'
[app         ] + '[' -n 700 ']'
[app         ] + log_message 'WINE: Enabling Virtual Desktop mode with 900:900 aspect ratio'
[app         ] ++ date
[app         ] + echo 'Mon 22 Apr 2024 07:27:57 PM UTC: WINE: Enabling Virtual Desktop mode with 900:900 aspect ratio'
[app         ] + winetricks vd=900x700
[app         ] ------------------------------------------------------
[app         ] Running Wine/winetricks as root is highly discouraged. See https://wiki.winehq.org/FAQ#Should_I_run_Wine_as_root.3F
[app         ] ------------------------------------------------------
[app         ] Executing mkdir -p /config
[supervisor  ] all services started.
[app         ] ------------------------------------------------------
[app         ] You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
[app         ] ------------------------------------------------------
[app         ] Using winetricks 20200412 - sha256sum: 7651c93e39fcb080483c38836513bf912273a87ea97d137f6b958ed3d9628c3d with wine-9.0 and WINEARCH=win64
[app         ] Executing wine regedit C:\windows\Temp/vd.reg
[app         ] Executing wine64 regedit C:\windows\Temp/vd.reg
[app         ] + '[' false = true ']'
[app         ] + '[' true = true ']'
[app         ] + echo 'FORCE_LATEST_UPDATE is enabled which may brick your installation.'
[app         ] FORCE_LATEST_UPDATE is enabled which may brick your installation.
[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + fetch_and_install
[app         ] + cd /config/wine/dosdevices/c:/
[app         ] + '[' true = true ']'
[app         ] + log_message 'INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] ++ date
[app         ] + echo 'Mon 22 Apr 2024 07:28:13 PM UTC: INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] + curl -L https://www.backblaze.com/win32/install_backblaze.exe --output install_backblaze.exe
[app         ]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[app         ]                                  Dload  Upload   Total   Spent    Left  Speed
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]  44 28.7M   44 12.8M    0     0  10.5M      0  0:00:02  0:00:01  0:00:01 10.5M
[app         ] 100 28.7M  100 28.7M    0     0  13.7M      0  0:00:02  0:00:02 --:--:-- 17.9M
[app         ] + log_message 'INSTALLER: Starting install_backblaze.exe'
[app         ] ++ date
[app         ] + echo 'Mon 22 Apr 2024 07:28:15 PM UTC: INSTALLER: Starting install_backblaze.exe'
[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + WINEARCH=win64
[app         ] + WINEPREFIX=/config/wine/
[app         ] + wine64 install_backblaze.exe
[xvnc        ] Mon Apr 22 19:28:50 2024
[xvnc        ]  Connections: accepted: /tmp/vnc.sock
[xvnc        ]  SConnection: Client needs protocol version 3.8
[xvnc        ]  SConnection: Client requests security type None(1)
[xvnc        ]  VNCSConnST:  Server default pixel format depth 24 (32bpp) little-endian rgb888
[xvnc        ]  VNCSConnST:  Client pixel format depth 24 (32bpp) little-endian bgr888

**Reply by kltpzyxM-retsiM** [reply]
Seeing same thing.  Looks like it gets hung trying to install BB.  Here are the logs I have in case it helps:  `text  error  warn  system  array  login  

[app         ] + for x in {d..z}
[app         ] + test -d /drive_d
[app         ] + for x in {d..z}
[app         ] + test -d /drive_e
[app         ] + for x in {d..z}
[app         ] + test -d /drive_f
[app         ] + for x in {d..z}
[app         ] + test -d /drive_g
[app         ] + for x in {d..z}
[app         ] + test -d /drive_h
[app         ] + for x in {d..z}
[app         ] + test -d /drive_i
[app         ] + for x in {d..z}
[app         ] + test -d /drive_j
[app         ] + for x in {d..z}
[app         ] + test -d /drive_k
[app         ] + for x in {d..z}
[app         ] + test -d /drive_l
[app         ] + for x in {d..z}
[app         ] + test -d /drive_m
[app         ] + for x in {d..z}
[app         ] + test -d /drive_n
[app         ] + for x in {d..z}
[app         ] + test -d /drive_o
[app         ] + for x in {d..z}
[app         ] + test -d /drive_p
[app         ] + for x in {d..z}
[app         ] + test -d /drive_q
[app         ] + for x in {d..z}
[app         ] + test -d /drive_r
[app         ] + for x in {d..z}
[app         ] + test -d /drive_s
[app         ] + for x in {d..z}
[app         ] + test -d /drive_t
[app         ] + for x in {d..z}
[app         ] + test -d /drive_u
[app         ] + for x in {d..z}
[app         ] + test -d /drive_v
[app         ] + for x in {d..z}
[app         ] + test -d /drive_w
[app         ] + for x in {d..z}
[app         ] + test -d /drive_x
[app         ] + for x in {d..z}
[app         ] + test -d /drive_y
[app         ] + for x in {d..z}
[app         ] + test -d /drive_z
[app         ] + cd /config/wine/
[app         ] + '[' false = true ']'
[app         ] + '[' -n 900 ']'
[app         ] + '[' -n 700 ']'
[app         ] + log_message 'WINE: Enabling Virtual Desktop mode with 900:900 aspect ratio'
[app         ] ++ date
[app         ] + echo 'Thu 25 Apr 2024 02:42:23 AM UTC: WINE: Enabling Virtual Desktop mode with 900:900 aspect ratio'
[app         ] + winetricks vd=900x700
[app         ] ------------------------------------------------------
[app         ] Running Wine/winetricks as root is highly discouraged. See https://wiki.winehq.org/FAQ#Should_I_run_Wine_as_root.3F
[app         ] ------------------------------------------------------
[app         ] Executing mkdir -p /config
[supervisor  ] all services started.
[app         ] ------------------------------------------------------
[app         ] You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
[app         ] ------------------------------------------------------
[app         ] Using winetricks 20200412 - sha256sum: 7651c93e39fcb080483c38836513bf912273a87ea97d137f6b958ed3d9628c3d with wine-9.0 and WINEARCH=win64
[app         ] Executing wine regedit C:\windows\Temp/vd.reg
[app         ] Executing wine64 regedit C:\windows\Temp/vd.reg
[app         ] + '[' false = true ']'
[app         ] + '[' true = true ']'
[app         ] + echo 'FORCE_LATEST_UPDATE is enabled which may brick your installation.'
[app         ] FORCE_LATEST_UPDATE is enabled which may brick your installation.
[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + fetch_and_install
[app         ] + cd /config/wine/dosdevices/c:/
[app         ] + '[' true = true ']'
[app         ] + log_message 'INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] ++ date
[app         ] + echo 'Thu 25 Apr 2024 02:42:25 AM UTC: INSTALLER: FORCE_LATEST_UPDATE=true - downloading latest version'
[app         ] + curl -L https://www.backblaze.com/win32/install_backblaze.exe --output install_backblaze.exe
[app         ]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[app         ]                                  Dload  Upload   Total   Spent    Left  Speed
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ]   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[app         ] 100 28.7M  100 28.7M    0     0  27.4M      0  0:00:01  0:00:01 --:--:-- 36.7M
[app         ] + log_message 'INSTALLER: Starting install_backblaze.exe'
[app         ] ++ date
[app         ] + echo 'Thu 25 Apr 2024 02:42:26 AM UTC: INSTALLER: Starting install_backblaze.exe'
[app         ] + '[' -f '/config/wine/drive_c/Program Files (x86)/Backblaze/bzbui.exe' ']'
[app         ] + WINEARCH=win64
[app         ] + WINEPREFIX=/config/wine/
[app         ] + wine64 install_backblaze.exe`

**Reply by kltpzyxM-retsiM** [reply]
Any luck figuring out the issue?  Any further debugs I can get you?

**Reply by traktuner** [reply]
Hey @kltpzyxM-retsiM 
since you chose to use the latest available Backblaze version (FORCE_LATEST_UPDATE=true), the latest version will be directly downloaded from Backblaze.
Can you test if you can download the file (https://www.backblaze.com/win32/install_backblaze.exe) directly on your docker host?
Because that official link is very reliable, and hardly ever offline. So if that's not working, there may be an internet or DNS issue with your docker setup (and troubleshooting would be very hard for us)

**Reply by stratriderz** [reply]
I was able to get the container functioning properly by completely removing it, deleting the underlying remaining files and reinstalling the container.  Thank you for the suggestions.

**Reply by cramsted** [reply]
For anyone else who encounters this issue who is on Unraid, I uninstalled the container and then deleted the cached files, and then reinstalled the container. I made sure to change the USER_ID and GROUP_ID to 0 at install time. Initially, I got the same screen as before, but I left it running for a few hours, and when I came back the first window had shown up. Subsequent steps took forever to progress through, but it seems like this problem can in part be solved with a little extra patience.
